includes the files 70-no-bitmaps-and-emoji.conf, 70-no-bitmaps-and-emoji.conf and
70-no-bitmaps.conf from Fedora extracted from fontconfig-2.15.0-6.fc41.x86_64.rpm
diff -rNu fontconfig-2.15.0/conf.d/70-no-bitmaps-and-emoji.conf fontconfig-2.15.0.new/conf.d/70-no-bitmaps-and-emoji.conf
--- fontconfig-2.15.0/conf.d/70-no-bitmaps-and-emoji.conf	1970-01-01 01:00:00.000000000 +0100
+++ fontconfig-2.15.0.new/conf.d/70-no-bitmaps-and-emoji.conf	2024-11-25 22:14:42.364803000 +0100
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<!DOCTYPE fontconfig SYSTEM "urn:fontconfig:fonts.dtd">
+<fontconfig>
+  <description>Reject bitmap fonts, including bitmap emoji fonts</description>
+  <!-- Reject bitmap fonts -->
+  <selectfont>
+    <rejectfont>
+      <pattern>
+        <patelt name="outline"><bool>false</bool></patelt>
+      </pattern>
+    </rejectfont>
+  </selectfont>
+</fontconfig>
diff -rNu fontconfig-2.15.0/conf.d/70-no-bitmaps-except-emoji.conf fontconfig-2.15.0.new/conf.d/70-no-bitmaps-except-emoji.conf
--- fontconfig-2.15.0/conf.d/70-no-bitmaps-except-emoji.conf	1970-01-01 01:00:00.000000000 +0100
+++ fontconfig-2.15.0.new/conf.d/70-no-bitmaps-except-emoji.conf	2024-11-25 22:14:42.364803000 +0100
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!DOCTYPE fontconfig SYSTEM "urn:fontconfig:fonts.dtd">
+<fontconfig>
+  <description>Reject bitmap fonts except bitmap emoji fonts</description>
+  <!-- Reject bitmap fonts -->
+  <selectfont>
+    <rejectfont>
+      <pattern>
+        <patelt name="outline"><bool>false</bool></patelt>
+        <patelt name="scalable"><bool>false</bool></patelt>
+      </pattern>
+    </rejectfont>
+  </selectfont>
+</fontconfig>
diff -rNu fontconfig-2.15.0/conf.d/70-no-bitmaps.conf fontconfig-2.15.0.new/conf.d/70-no-bitmaps.conf
--- fontconfig-2.15.0/conf.d/70-no-bitmaps.conf	2023-12-22 14:43:03.000000000 +0100
+++ fontconfig-2.15.0.new/conf.d/70-no-bitmaps.conf	2024-11-25 22:14:42.364803000 +0100
@@ -1,13 +1,6 @@
 <?xml version="1.0"?>
 <!DOCTYPE fontconfig SYSTEM "urn:fontconfig:fonts.dtd">
 <fontconfig>
-  <description>Reject bitmap fonts</description>
-<!-- Reject bitmap fonts -->
- <selectfont>
-  <rejectfont>
-   <pattern>
-     <patelt name="outline"><bool>false</bool></patelt>
-   </pattern>
-  </rejectfont>
- </selectfont>
+  <description>Reject bitmap fonts except bitmap emoji fonts (deprecated; use 70-no-bitmaps-except-emoji.conf)</description>
+  <include ignore_missing="yes">70-no-bitmaps-except-emoji.conf</include>
 </fontconfig>
diff -rNu fontconfig-2.15.0/conf.d/meson.build fontconfig-2.15.0.new/conf.d/meson.build
--- fontconfig-2.15.0/conf.d/meson.build	2023-12-22 14:43:03.000000000 +0100
+++ fontconfig-2.15.0.new/conf.d/meson.build	2024-11-25 22:56:31.015982512 +0100
@@ -36,6 +37,8 @@
   '65-nonlatin.conf',
   '69-unifont.conf',
   '70-no-bitmaps.conf',
+  '70-no-bitmaps-and-emoji.conf',
+  '70-no-bitmaps-except-emoji.conf',
   '70-yes-bitmaps.conf',
   '80-delicious.conf',
   '90-synthetic.conf',
